codegens/nodejs-native/test/unit/snippet.test.js

```diff
@@ -175,4 +175,39 @@ describe('nodejs-native convert function', function () {
       expect(snippet).to.include('name=\\"invalid src\\"; filename=\\"file\\"');
     });
   });
+  it('should generate valid snippet for single/double quotes in url', function () {
+    var request = new sdk.Request({
+      'method': 'GET',
+      'header': [],
+      'url': {
+        'raw': 'https://postman-echo.com/get?query1=b\'b&query2=c"c',
+        'protocol': 'https',
+        'host': [
+          'postman-echo',
+          'com'
+        ],
+        'path': [
+          'get'
+        ],
+        'query': [
+          {
+            'key': 'query1',
+            'value': 'b\'b'
```

Comment:
why is the single quote already escaped?
If you use double quotes around it and do `// eslint-disable`, will the test pass?

Reply:
changed.